lib/components/narrative/default/default-itinerary.js

```diff
@@ -132,12 +128,43 @@ const ITINERARY_ATTRIBUTES = [
         // FIXME: For CAR mode, walk time considers driving time.
         <>
           <FormattedDuration duration={itinerary.walkTime} />
-          <LegIconWrapper>
+          <LegIconWrapper noSpace>
             <LegIcon leg={leg} size={5} />
           </LegIconWrapper>
         </>
       )
     }
+  },
+  {
+    id: 'carbonEmission',
+    order: 4,
+    render: (itinerary, { co2VsBaseline }) => {
+      if (!co2VsBaseline) return
+      const ROUND_TO_NEAREST = 5
+      co2VsBaseline =
+        Math.round((co2VsBaseline * 100) / ROUND_TO_NEAREST) * ROUND_TO_NEAREST
+      if (Math.abs(co2VsBaseline) <= 10) return // Only show greater than 10%
```

Comment:
Should this maybe be configurable? I would also set this to something like 33% or something so it doesn't show up as often

Reply:
Done!